public/api/conf/1.0/docs/schema.md

```diff
@@ -0,0 +1 @@
+Schema definitions for all the API endpoints request & response payloads can be found in the following [schemas and example messages](/api-documentation/docs/api/download/customs-declarations/1.0/wco-declaration-schemas.zip) zip archive.
```

Comment:
That link is for dec service. You need a link for dec info. Change service name and see build.sbt.